UpdatedImageProcessing/test3.py

```diff
@@ -0,0 +1,15 @@
+d = {
+	"a": 0,
+	"b": 1,
+	"c": 2
+}
+
+print("before")
+
+print(d)
+
+del d[min(d, key=d.get)]
+
+print("after")
+
+print(d)
```

Comment:
Please remove this test file.

Reply:
Removed in https://github.com/FlintHill/SUAS-Competition/pull/97/commits/bfde9a153d5b899db8812b478525ed8542de8f28 & https://github.com/FlintHill/SUAS-Competition/pull/97/commits/a81de5d170d31a737251567f72ec9961d3b3bd9e